actions/checks.deleted.machines.yaml

```diff
@@ -0,0 +1,25 @@
+description: Check For machines stuck in the deleting state
+enabled: true
+entry_point: src/openstack_check_actions.py
+name: checks.deleted.machines
+parameters:
+  submodule:
+    default: deleting_machines_check
+    immutable: true
+    type: string
+  cloud_account:
+    description: The clouds.yaml account to use whilst performing this action
+    default: ""
+    required: true
+    type: string
+  project_id:
+    description: ID of the project to be scanned for the above security group details (*Requires one of)
+    default: ""
+    required: false
+    type: string
+  all_projects:
+    description: Toggle to scan all projects in cloud (*Requires one of)
```

Comment:
We can drop `Requires one of` since the GUI/API will enforce the bool type

Reply:
This is more related too needing one of either project ID or a True for all projects. Is there a better way I could enforce that?

Reply:
Ah okay, I assumed it was a partial sentence. Now I understand it I think this is fine.

As long as we have a test, something like 

```
if bool(project_id) == all_projects:
    raise RuntimeError("A project ID or all projects must be enabled. Either both or neither were entered")
```